src/@openarchitex/gatsby-theme-minimal-portfolio/components/footer.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faFacebook, faLinkedin } from '@fortawesome/free-brands-svg-icons'
+import ListGroup from "react-bootstrap/ListGroup";
+
+const Footer = () => (
+    <footer>
+        <div className="container-fluid">
+            <div className="mt-3 text-center">
+                <div className="row">
+                    <div className="mx-auto">
+                        <ListGroup horizontal>
+                            <ListGroup.Item className="border-0"><a aria-label="facebook" style={{color: "inherit"}} href="https://www.facebook.com/OpenArchitex-588167811850853/" target="_blank" rel="noopener noreferrer"><FontAwesomeIcon icon={faFacebook} size="2x"/></a></ListGroup.Item>
+                            <ListGroup.Item className="border-0"><a aria-label="instagram" style={{color: "inherit"}} href="https://www.linkedin.com/company/openarchitex/" target="_blank" rel="noopener noreferrer"><FontAwesomeIcon icon={faLinkedin} size="2x"/></a></ListGroup.Item>
```

Comment:
@faithghlee : the `aria-label` needs to be changed to `linkedin`. This is for screen-readers and stuff.